pypeit/core/wavecal/autoid.py

```diff
@@ -32,6 +33,9 @@
 
 from matplotlib import pyplot as plt
 from matplotlib import gridspec
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import matplotlib.colorbar as colorbar
+import matplotlib.colors as mcolors
```

Comment:
I'd prefer

```
from matplotlib import colorbar
import matplotlib.colors
```
and then replace `mcolors` with `matplotlib.colors` in the code.

Reply:
Good idea - done!